joint_limits:
  wheel_joint_left:
    has_acceleration_limits: true
    max_acceleration: 55 #rad*s^(-2)
  wheel_joint_right:
    has_acceleration_limits: true
    max_acceleration: 55 #rad*s^(-2)

mrvk_joint_names: [wheel_joint_left, wheel_joint_right, scan_to_veza, camera_r_to_camera]
#[main_board motor_control_board_left motor_control_bord_right]
port_names: [/dev/MB, /dev/MCBL, /dev/MCBR]

baudrate: 230400
stop_bits: 1    #1 - stopBits 1  #2 - stopBits 2 #3 - stopBits 1.5 
parity: 1       #0 - none #1 - odd parity #2 - even parity
byte_size: 8    #5 - fivebits #6 - sixbits #7 - sevenbits #8 eightbits
 
status_period: 5

mcb_5V: True
mcb_12V: True
fan: Trues
video_transmitter: True #spustenie kinectu
laser_scanner: True
gps: False
pc2: False
camera: True
arm_5V: False
arm_12V: False
video: True      #true obraz z kamery na grabbery a z ramena na transmitter #false opacne
 
left_P: 10
left_I: 15
left_pwm_control: False   #false - otackova regulacia #true - PWM

right_P: 10
right_I: 15
right_pwm_control: False   #false - otackova regulacia #true - PWM

camera_pko: 5
camera_pkk: 10
camera_iko: 5
camera_ikk: 80

comunication_rate: 50 # 30 je asi max (default 30)
